src/css/main.css

```diff
@@ -587,8 +587,8 @@ span.subcomponent {
   cursor: grabbing;
 }
 
-code, pre {
```

Comment:
no, you don't want `Roboto Mono` in both instances. because, the whole point of the Google Web Font Loader is to allow you to not have the white Flash Of Invisible Text ([FOIT](https://css-tricks.com/fout-foit-foft/)). but you have to use the classes (e.g., `wf-active`) that are set by the Web Font Loader that tell you when `Roboto Mono` has loaded. otherwise, the browser will wait for `Roboto Mono` to load before rendering _any_ text. so the approach would be to have all these lines:

```css
code,
pre,
input,
textarea,
select {
  font-family: Consolas, Andale Mono, Monaco, Courier New, monospace;
}

.wf-active code,
.wf-active pre,
.wf-active input,
.wf-active textarea,
.wf-active select {
  font-family: Roboto Mono, Consolas, Andale Mono, Monaco, Courier New, monospace;
}
```

hope that makes sense.